### ISSUE TITLE
Conflict with CompressionWebpackPlugin

### ISSUE THREAD

**Issue by ntkoso** [vigneshshanmugam/optimize-js-plugin#4]
**Config:**

``` js
import OptimizeJsPlugin from 'optimize-js-plugin';
import CompressionWebpackPlugin from 'compression-webpack-plugin';

plugins: [
  new webpack.optimize.UglifyJsPlugin(options),
  new OptimizeJsPlugin({
    sourceMap: false
  }),
  new CompressionWebpackPlugin(options)
]
```

**Current behavior:**
Chunks are getting optimized, but compressed files contain original `uglifyjs` output without optimizations.

**Expected/desired behavior:**
Compressed files should contain optimized chunks instead of originals.
`UglifyJsPlugin -> OptimizeJsPlugin -> CompressionWebpackPlugin`

**Comment by vigneshshanmugam** [comment]
@ntkoso Thanks for finding it, seems to be a bug. Will have a look.

**Comment by vigneshshanmugam** [comment]
@ntkoso fixed it in 0.0.4
